helm/values-dev.yaml

```
@@ -80,8 +79,9 @@ microservice-chart:
     # required
     APPLICATIONINSIGHTS_CONNECTION_STRING: 'ai-d-connection-string'
     OTEL_EXPORTER_OTLP_HEADERS: elastic-apm-secret-token
+    MONGODB_CONNECTION_URI: 'notices_mongo_connection_string'
   keyvault:
-    name: "pagopa-d-name-kv" #TODO
+    name: "pagopa-d-printit-kv"
```

Diego decided to put `itn` in the name
 
```suggestion
    name: "pagopa-d-itn-printit-kv"
```